client/modules/accounts/templates/signUp/signUp.js

```diff
@@ -61,13 +62,27 @@ Template.loginFormSignUpView.events({
       return;
     }
 
+    const accountDetails = {
+      userId: Meteor.userId(),
+      role: role
+    };
+
+    // adds role to user account
+    Meteor.call('user/addUserRole', accountDetails, (err, res) => {
+      if (err) {
+        alert(err);
+      } else {
+        // success!
+      }
+    });
+
     const newUserData = {
-      // username: username,
       email: email,
-      password: password
+      password: password,
+      role: role
     };
 
-    Accounts.createUser(newUserData, function (error) {
+    Accounts.createUser(newUserData, function(error) {
```

Comment:
"arrowFunctions": true,

Please lint and see if linter picks up the .eslintrc settings